content/auth/capabilities.textile

```diff
@@ -353,6 +379,32 @@ if err != nil {
 // }
 ```
 
+```[flutter]
+// API key capabilities:
+// {
+//   'chat:*': ['publish', 'subscribe', 'presence'],
+//   'status': ['subscribe', 'history'],
+//   'alerts': ['subscribe']
```

Comment:
Maybe it's better to use capabilities from code below?

```suggestion
//   'chat:bob': ['subscribe'],
//   'status': ['*'],
//   'secret': ['publish', 'subscribe']
```

Reply:
Agreed! Have updated them.